docker-compose.yaml

```diff
@@ -4,11 +4,32 @@ services:
     build: .
     image: mine-support-claim-service
     container_name: mine-support-claim-service
+    command: sh -c "npx wait-on -v tcp:mine-support-claim-postgres:5432 && npm run migrate && npm run start:watch"
+    depends_on:
+      - mine-support-claim-postgres
+    environment:
+      POSTGRES_DB: mine_claims
+      POSTGRES_PASSWORD: ppp
+      POSTGRES_USERNAME: postgres
+      POSTGRES_HOST: mine-support-claim-postgres
+      POSTGRES_PORT: 5432

```

Comment:
This should also include environment variables for the message queue

Reply:
Added the message queue environment variables